test/unit/color/p5.Color.js

```diff
@@ -781,7 +781,7 @@ suite('p5.Color', function() {
     });
   });
 
-  suite.only('p5.Color.prototype.toString', function() {
+  suite('p5.Color.prototype.toString', function() {
```

Comment:
Looks like this was introduced in #1463. It didn't seem to affect running tests from the command-line, but when run in-browser it'd configure mocha to [only run this suite](http://jaketrent.com/post/run-single-mocha-test/) and no others.